pkg/create/templates/docker.go

```diff
@@ -3,38 +3,56 @@ package templates
 // Dockerfile is a default Dockerfile laid down by s2i create
 const Dockerfile = `
 # {{.ImageName}}
-FROM openshift/base-centos7
```

Comment:
If there's a middle ground where a developer starting from a bootstrap template just has to change lines like:
```
# TODO you need to change this
ENTRYPOINT nginx --arg=foo
```
Then I think it's a good plan.  The thing I'm worried about with this PR is that they would be supposed to rip out additional nginx-specific boilerplate which shouldn't be in a generic bootstrap template (but fits OK in an example).